cmd/mongodb-healthcheck/logger/logger.go

```diff
@@ -0,0 +1,96 @@
+package logger
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/go-logr/logr"
+	uzap "go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
```

Comment:
Shouldn't we use `"github.com/percona/percona-server-mongodb-operator/cmd/mongodb-healthcheck/logger/lumberjack"`here?

Reply:
Just noticed we are replacing the dependency with the local directory. We are good then!